delta-lake/common/src/main/delta-io/scala/com/nvidia/spark/rapids/delta/DeltaIOProvider.scala

```diff
@@ -95,12 +96,12 @@ abstract class DeltaIOProvider extends DeltaProviderImplBase {
       writeOptionsFromExec(cpuExec.writeOptions), cpuExec.session)
   }
 
-  private case class DeltaWriteV1Config(
+  protected case class DeltaWriteV1Config(
       deltaLog: DeltaLog,
       forceOverwrite: Boolean,
       options: mutable.HashMap[String, String])
 
-  private def extractWriteV1Config(
+  protected def extractWriteV1Config(
```

Comment:
why is the method `protected` here and `private` in Databricks. 


```suggestion
  private def extractWriteV1Config(
```